src/endpoints/seed/index.ts

```diff
@@ -838,40 +821,59 @@ export const seed = async ({
     ),
   )
 
-  payload.logger.info(`— Seeding globals...`)
-
-  await Promise.all([
-    payload.updateGlobal({
-      slug: 'footer',
-      data: {
-        navItems: [
-          {
-            link: {
-              type: 'custom',
-              label: 'Admin',
-              url: '/admin',
-            },
-          },
-          {
-            link: {
-              type: 'custom',
-              label: 'Source Code',
-              newTab: true,
-              url: 'https://github.com/payloadcms/payload/tree/main/templates/website',
-            },
-          },
-          {
-            link: {
-              type: 'custom',
-              label: 'Payload',
-              newTab: true,
-              url: 'https://payloadcms.com/',
-            },
-          },
-        ],
+  payload.logger.info(`— Seeding footers...`)
+  const footerData: Record<Tenant['slug'], {}> = {
```

Comment:
Could the values of this record be typed with `RequiredDataFromCollectionSlug<'footer'>`?

Reply:
Thanks - I could not figure out the type for this one

Reply:
```js
src/endpoints/seed/index.ts(846,5): error TS2322: Type '{ address: string; email: string; socialMedia: {}; contentHash: null; }' is not assignable to type 'Omit<Footer, "createdAt" | "updatedAt" | "sizes" | "id"> & Partial<Pick<Footer, "createdAt" | "updatedAt" | "sizes" | "id">>'.
  Type '{ address: string; email: string; socialMedia: {}; contentHash: null; }' is missing the following properties from type 'Omit<Footer, "createdAt" | "updatedAt" | "sizes" | "id">': name, tenant
  ```

Reply:
Ah, I see - unless you need to have the proto-data in its own variable for something later, this got worked around in the other calls to `upsert` by just inlining the data and mapping functions. No biggie either way, as it looks like all the fields being spread by `...footerData[tenant.slug]` are optional anyway.